frontend/templates/troubleshooting/index.tsx

```diff
@@ -411,6 +410,7 @@ function NavBar({
    const breadcrumbMinHeight = '48px';
    return (
       <Flex
+         className="NavBar"
```

Comment:
Why add this className?

Reply:
Classes make it super-easy to find the component when modifying styles. I've been using the component name Mattia uses in Figma to keep that association as well.

Reply:
I've been wondering if there is a browser extension that can replace classnames with the component name because the auto generated chakra classes are hard to read through.

Reply:
I wish that was a thing. That's pretty much exactly the issue I'm trying to reduce. I'm tired of having to hunt for a unique parent attribute, and drill down from there.